mmstudio/src/main/java/org/micromanager/display/internal/displaywindow/DisplayController.java

```diff
@@ -526,8 +526,8 @@ public void run() {
    protected Coords handleDisplayPosition(Coords position) {
       perfMon_.sampleTimeInterval("Handle display position");
 
-      runnablePool_.invokeLaterWithCoalescence(
-            new ExpandDisplayRangeCoalescentRunnable(position));
+      //runnablePool_.invokeLaterWithCoalescence(
+      //      new ExpandDisplayRangeCoalescentRunnable(position));
```

Comment:
It would be nice to avoid commented code (old code is kept in git!).

In the original way I expected this code to work, removing this would be a bug (same as omitting the `animationAcknowledgeDataPosition()` firing elsewhere), but is there a mechanism that replaces this now? If so, the 2 lines should just be deleted. If updates to the display range (i.e. the display's model of the size of the datastore) are now handled without coalescent runnables, then it would probably be cleanest to remove all related code. If there is no such mechanism, then I'm pretty sure these lines are needed, though I'd be happy to be proven wrong.

Reply:
Just tested, and leaving this code in slows down the display to the point of it becoming unworkable.  The display range is updated in the DisplayUIController.displayImage function.  Not sure about code paths that do not call displayImage (if there are such updates to the viewer without displaying an image).